package.json

```diff
@@ -25,11 +25,12 @@
   "scripts": {
     "start": "craco start",
     "build": "craco build",
-    "build:milkomeda-testnet": "npx -y env-cmd -f ./env/milkomeda-testnet.env npm run build",
+    "build:milkomeda-c1-testnet": "npx -y env-cmd -f ./env/milkomeda-testnet.env npm run build",
```

Comment:
Let's get rid of the "c1" here.

Reply:
Why did you mark this as resolved, but the "c1" is still there?

Reply:
My bad, instead of changing the package.json, I changed the names of ci pipeline to fix build issues. I wasn't aware of the fleek naming convention. I will change it in the next commit.